templates/components/settings/python/settings.py

```
@@ -105,34 +114,50 @@
     # Groq does not provide embeddings, so we use FastEmbed instead
     init_fastembed()
 
 
 def init_anthropic():
-    from llama_index.llms.anthropic import Anthropic
+    try:
+        from llama_index.llms.anthropic import Anthropic
+    except ImportError:
+        raise ImportError(
+            "Anthropic support is not installed. Please install it with `pip install llama-index[anthropic]`"
+        )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)